### ISSUE TITLE
Automatically stop when improvement slows?

### ISSUE THREAD

**Issue by Collinthegeek** [jcjohnson/torch-rnn#171]
I'd like an option to stop training when it's plateaued.

When loss stays effectively the same for X amount of time/cycles

**Comment by dgcrouse** [comment]
Loss can plateau then decrease again as learning rate varies, this is very common in deep learning. I'm not sure this will be a useful feature to add because of this.
